modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -58,26 +60,61 @@
         $instrument = $_POST['instrument'] !== '' ? $_POST['instrument'] : null;
         $pscid      = $_POST['pscid']      !== '' ? $_POST['pscid'] : null;
         $visit      = $_POST['visit']      !== '' ? $_POST['visit'] : null;
-        $comments   = $_POST['comments']   !== '' ? $_POST['commnets'] : null;
+        $comments   = $_POST['comments']   !== '' ? $_POST['comments'] : null;
         $version    = $_POST['version']    !== '' ? $_POST['version'] : null;
+        $uuid       = uuid4();
 
         $fileSize = $_FILES["file"]["size"];
         $fileName = $_FILES["file"]["name"];
-        $fileType = end((explode(".", $fileName)));
+        $fileType = '';
+        // Handle retrieving the file type.
+        if (preg_match('/\./', $fileName)) {
+            $pos = strrpos($fileName, '.', -1);
+            if ($pos+1 != strlen($fileName)) {
+                $fileType = substr(
+                    $fileName,
+                    strrpos($fileName, '.', -1)+1
+                );
+            }
+        }
+        $sql_statement = $db->prepare(
+            'SELECT File_name, version FROM document_repository '
+            .'WHERE File_name=? AND uploaded_by=?'
+        );
+        $sql_statement->bindParam(1, $fileName, PDO::PARAM_STR);
+        $sql_statement->bindParam(2, $puser, PDO::PARAM_STR);
+        $sql_statement->execute();
+        $sql_result = $sql_statement->fetchAll(PDO::FETCH_ASSOC);
 
         // __DIR__ is the document_repository ajax directory
         // when this script is executing. Go up a level to the
         // document_repository module directory, and use a
         // user_uploads directory as a base for user uploads
-        $base_path = __DIR__ . "/../user_uploads/";
-        $fileBase  = $puser . "/" . $fileName;
+        $base_path = realpath(__DIR__ . '/..') . '/user_uploads/';
```

Comment:
Would you mind changing this to `$upload_path`? The "base path" in LORIS typically refers to the Config setting corresponding to the web root.